frameworks/Python/uvicorn/uvicorn_conf.py

```diff
@@ -0,0 +1,13 @@
+import multiprocessing
+import os
+
+_is_travis = os.environ.get('TRAVIS') == 'true'
+
+workers = multiprocessing.cpu_count()
+if _is_travis:
+    workers = 2
+
+bind = "0.0.0.0:8080"
+keepalive = 120
+errorlog = '-'
+pidfile = 'uvicorn.pid'
```

Comment:
Sure thing. Few questions off that:

* We could also alternatively avoid writing a pid file completely, if you prefer?
* Right now I'm just following the convention of other Python frameworks, which [write an equivalent `gunicorn.pid`](https://github.com/TechEmpower/FrameworkBenchmarks/search?utf8=%E2%9C%93&q=gunicorn.pid). I'd be happy to pull together a pull request changing those too, if that's helpful?
* Is either of `/tmp` or `$IROOT` preferable to the other?